.spelling

```diff
@@ -582,6 +589,9 @@ user
 user1
 UTF-8
 v1
+v1.13
+v1.14
+v1.15
```

Comment:
I don't think these are needed, In the docs we usually say Kubernetes releases 1.13, 1.14, 1.15.

Reply:
how about not modifying the `.spelling` file at all, and then we can instruct exactly what needs to be added to .spelling. 

The `.spelling` file is a list of exceptions. Exceptions should not be an exception, not the norm.

Reply:
> how about not modifying the `.spelling` file at all, and then we can instruct exactly what needs to be added to .spelling.
> 
> The `.spelling` file is a list of exceptions. Exceptions should not be an exception, not the norm.

@sdake 

Agree. I only add [two words](https://github.com/istio/istio.io/pull/6424/files#diff-b9a2754b5f195f0113dc72bd305fcdb3R337) for exceptions, the above changes are outdated.